pkg/k8s/synced/crd.go

```diff
@@ -42,7 +42,6 @@ func agentCRDResourceNames() []string {
 		CRDResourceName(v2.CCNPName),
 		CRDResourceName(v2.CNName),
 		CRDResourceName(v2.CIDName),
-		CRDResourceName(v2.CIDName),
 		CRDResourceName(v2alpha1.BGPPName),
 		CRDResourceName(v2alpha1.BGPPoolName),
```

Comment:
These 2 lines need to be removed otherwise the agent will be waiting for the resources to be synced.